vite.config.ts

```diff
@@ -34,6 +38,18 @@ export const config: UserConfigFn = ({ mode }) => {
         },
       }),
       svgLoader(),
+      yamlLoader(
```

Comment:
**Issue** (minor): Not exactly sure where this happens, but in this PR, the YAML rendered on page now has an extra empty line at the end which shouldn’t be there.

The rest looks good to me.

Reply:
Oh good spot, lemme take a look

Reply:
trimmed things